ACadSharp/IO/Templates/CadLeaderTemplate.cs

```diff
@@ -7,7 +8,24 @@ internal class CadLeaderTemplate : CadEntityTemplate
 		public CadLeaderTemplate(Leader entity) : base(entity) { }
 
 		public double Dimasz { get; internal set; }
+
 		public ulong DIMSTYLEHandle { get; internal set; }
+
 		public ulong AnnotationHandle { get; internal set; }
+
+
+		public override void Build(CadDocumentBuilder builder)
+		{
+			base.Build(builder);
+
+			Leader leader = (Leader)this.CadObject;
+
+			if (this.getTableReference(builder, this.DIMSTYLEHandle, string.Empty, out DimensionStyle style)) {
+				leader.Style = style;
+			}
+			if (builder.TryGetCadObject(this.AnnotationHandle, out Entity annotation)) {
+				leader.AssociatedAnnotation = annotation;
+			}
```

Comment:
The annotation is always present? if it is, it would be good to have an else with a log using the notification method so the user knows why the annotation is missing when the reader finishes.

Reply:
you provided the `getTableReference` method for table references. Shoud we implement a similar method for Entity references?

Reply:
I'm not 100% sure but I think that some entity references may have a null value, so I'm not sure that this method needs to be implemented, that's why I'm asking if the AssociatedAnnotation can be null or not, because if it can, there is no need to notify anything.

Reply:
We checked it with AutoCAD:
It is possible to create a LEADER without annotation. Then the field does not appear in DXF.
In this case the `AssociatedAnnotation` property should be null.

Reply:
Exactly, that's why I've asked if this parameter is optional, because if it is, not finding the value or assigning a null value is a valid resolution for the reference and should not be notified as an error or warning.